lldb/source/Commands/CommandObjectDWIMPrint.cpp

```diff
@@ -61,6 +63,36 @@ void CommandObjectDWIMPrint::HandleArgumentCompletion(
 
 bool CommandObjectDWIMPrint::DoExecute(StringRef command,
                                        CommandReturnObject &result) {
+  // BEGIN SWIFT
+  // Dump the contents of the valobj to result, possibly prepending it with
+  // a note telling the user that p is potentially better.
+  auto write_valobj_to_result =
+      [&](ValueObject &valobj, const DumpValueObjectOptions &dump_options) {
```

Comment:
you could make this a static function on one of the commands, to allow the other commands to re-use it.